mlir/include/mlir-c/Dialect/Rock.h

```diff
@@ -140,6 +133,33 @@ bool mlirRockTuningSetFromTable(MlirRockTuningTable perfTable,
 MLIR_CAPI_EXPORTED size_t mlirRockTuningGetKey(MlirModule module, char *buf,
                                                size_t bufLen);
 
+// TODO (ravil): document
+MLIR_CAPI_EXPORTED
+enum RocmlirSplitKSelectionLikelihood
+mlirIsSplitKFaster(int64_t gDim, int64_t mDim, int64_t nDim, int64_t kDim,
+                   int64_t numCUs, RocmlirTuningParamSetKind tuningLevel);
+
+// TODO (ravil): document
+MLIR_CAPI_EXPORTED
+bool mlirIsModuleFusible(MlirModule module, MlirStringRef perfStr);
+
+MLIR_CAPI_EXPORTED
+size_t mlirGetNumPrefillArgs(MlirModule module);
+
+// TODO (ravil): document
+MLIR_CAPI_EXPORTED
+void mlirGetPrefillArgsInfo(MlirModule module, size_t *indices,
```

Comment:
For my own paranoia, make this take a `size_t` giving the lengths of those arrays.

Reply:
take the sizes of array and add an assert